examples/apps/kubernetes-dashboard/README.md

```diff
@@ -0,0 +1,6 @@
+```sh
+
+kubectl proxy
+http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/#/login
+
+```
```

Comment:
Should we add k8s dashboard here? It doesn't interact with Busola in any meaningful way, except for adding some resources we can see in our UI.

Reply:
I don't have an opinion about this. We can remove it